### PR TITLE
Update domain_check.py

### PR DESCRIPTION
**Pull request by superpoussin22**
make url RFC 1808 compliant if it did not start with // or http:// or https://

### Describe the Pull Request

```
if url is not RFC 1808 compliant it won't be analyzed by malware analyzer 
```

### Checklist for PR

- [X] Run MobSF unit tests (http://your-mobsf-@ip:8000/tests/ or python3 manage.py test).
- [X] Tested Working on Linux, Mac, Windows, and Docker
- [X] Make sure build is passing on your PR. [Build Status](https://travis-ci.com/MobSF/Mobile-Security-Framework-MobSF/pull_requests)
- [X] Coding style (indentation, etc)

### CODE REVIEW

**Comment by superpoussin22** [comment]
perpas we could sanityze url received from malwarelist cause there's some entries with www. :(

File: MalwareAnalyzer/views/domain_check.py [hunk 1]
```diff
@@ -105,7 +105,12 @@ def get_domains(urls):
         domains = []
         for url in urls:
             parsed_uri = urlparse(url)
+            if not urlparse(url).scheme:
+                url = "//"+url
```
Comment (ajinabraham) [inline MalwareAnalyzer/views/domain_check.py]: This is good!

File: MalwareAnalyzer/views/domain_check.py [hunk 2]
```diff
@@ -105,7 +105,12 @@ def get_domains(urls):
...
+                parsed_uri = urlparse(url)
             domain = '{uri.netloc}'.format(uri=parsed_uri)
+            if domain.startswith('www.'):
```
Comment (ajinabraham) [inline MalwareAnalyzer/views/domain_check.py]: This is a problem. We have malware signature with `www.` and that will break it.

Comment (ajinabraham) [reply MalwareAnalyzer/views/domain_check.py]: Can work if you change statswith to contains in here: https://github.com/MobSF/Mobile-Security-Framework-MobSF/blob/master/MalwareAnalyzer/views/domain_check.py#L86

Comment (superpoussin22) [reply MalwareAnalyzer/views/domain_check.py]: corrected :)

**Comment by ajinabraham** [comment]
I need to do more testing with this. I will get back.

File: MalwareAnalyzer/views/domain_check.py [hunk 3]
```diff
@@ -83,7 +83,9 @@ def malware_check(urllist):
                     details_dict["desc"] = enlist[4]
                     details_dict["bad"] = "yes"
                     for domain in domainlist:
-                        if (details_dict["domain_or_url"].startswith(domain) or
+                        if ((details_dict["domain_or_url"].startswith(domain) or
+                                (domain.startswith(details_dict["domain_or_url"], 4) and
+                                 len(details_dict["domain_or_url"]) > 1)) or
```
Comment (ajinabraham) [inline MalwareAnalyzer/views/domain_check.py]: I think a better code will be 

```if domain in details_dict["domain_or_url"] or details_dict["domain_or_url"] in domain```

Comment (superpoussin22) [reply MalwareAnalyzer/views/domain_check.py]: will test :) need to ensure we will not match domain containing a - cause the malware domain list  contain a - for domain when there's only an IP as info

Comment (superpoussin22) [reply MalwareAnalyzer/views/domain_check.py]: like this it works : 
`if domain in details_dict["domain_or_url"] or 
    (details_dict["domain_or_url"] in domain and (len(details_dict["domain_or_url"]) > 1)) or 
    details_dict["ip"].startswith(domain):`

**Comment by superpoussin22** [comment]
@ajinabraham yes it will

**Pull request merged**